sdk-base/src/main/java/com/mapbox/maps/extension/observable/eventdata/CameraChangedEventData.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.maps.extension.observable.eventdata
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * The data class for camera-changed event data in Observer
+ */
+data class CameraChangedEventData(
```

Comment:
oh interesting so it does have fields

Reply:
yes, all the events have the begin field and optional end field.